packages/main/src/DateComponentBase.ts

```diff
@@ -125,6 +133,24 @@ class DateComponentBase extends UI5Element {
 		return this._formatPattern !== "medium" && this._formatPattern !== "short" && this._formatPattern !== "long";
 	}
 
+	_getMinMaxCalendarDateFromString(date: string) {
```

Comment:
i would remove that `return null` because with it the method have 3(CalendarDate | undefined | null) return types instead of 2 (CalendarDate | undefined)